manifest.yml

```diff
@@ -0,0 +1,64 @@
+# Step 1: Setup a cloud.gov account, follow instructions to install the cf-cli, and login on the command line:
```

Comment:
unfortunately not without significant documentation and flag usage, if the file namespace doesn't conflict with the overall project I'd like to keep it to avoid potential deployment confusion